onboardingtaskapp.client/src/components/Store/Create.jsx

```diff
@@ -24,20 +30,35 @@ export default function Create() {
     const handleSubmit = async (e) => {
         e.preventDefault();
         try {
-            const response = await fetch('https://localhost:7207/api/Store', {
+            const response = await fetch('https://onboardsite.azurewebsites.net/api/Store', {
```

Comment:
1. Please check my previous comment about console.logs
2. Please implement the API endpoint that I mentioned above